### ISSUE TITLE
Uknown padding left in landscape mode

### ISSUE THREAD

**Issue by PlinioMayer** [kempsteven/vue-html2pdf#22]
First of all I would like to thank you for your software, it actually saved me in my work. I had a great time generating pdf's in portrait mode, no problems. But when I try to generate in landscape, the layout preview looks fine, but the generated pdf comes with a padding left that I can't get rid off

`<vue-html2pdf
      :show-layout="true"
      :enable-download="true"
      :preview-modal="true"
      :paginate-elements-by-height="3000"
      filename="file"
      :pdf-quality="2"
      pdf-format="a4"
      pdf-orientation="landscape"
      pdf-content-width="1123px"
      @hasGenerated="$emit('generated', $event)"
    >`

The layout I'm passing is just a section with width 100% and height 794px aligning some text. Some images:

Layout:

![layout](https://user-images.githubusercontent.com/55549580/89044645-9d518100-d320-11ea-9e88-238a6119e276.PNG)

Preview:

![preview](https://user-images.githubusercontent.com/55549580/89044714-bb1ee600-d320-11ea-9bac-516b4b05cf32.PNG)

**Comment by kempsteven** [comment]
Hey thanks for appreciating my work!

I tried replicating what you did from your example, Here is my code:
![image](https://user-images.githubusercontent.com/51051704/89092180-1e248180-d3e2-11ea-9cee-8b6a82a219a2.png)

and here is the result, I can't seem to replicate your issue.
![image](https://user-images.githubusercontent.com/51051704/89092200-4318f480-d3e2-11ea-958d-66e32bab7c59.png)

But when I added `margin-left` on the element, it seems to have that white space on the left like from your screenshot, so there could be margin on one of your elements I presume? could you give me a sample your styling? ill try it on my end.

**Comment by kempsteven** [comment]
Feel free to open this if the issue persist!

**Comment by akbarism** [comment]
i have same issue here,

**Comment by cassioutfpr** [comment]
I had a similiar issue here but in portrait mode. I checked my elements and none of them seems to have a margin left (of course that I can be wrong).
But this is my code:

![image](https://user-images.githubusercontent.com/9342675/133289386-abc84f76-24cd-4da3-9233-9a21a5564c93.png)

And this is the result:

![image](https://user-images.githubusercontent.com/9342675/133289450-9bfa134b-713e-46d1-8c86-5db972b01dae.png)


I fixed it by appying a margin-left: -10px in the inner section component. I'm using vue 2.
 btw thanks for this repo!
